contracts/CollateralTracker.sol

```diff
@@ -134,15 +134,15 @@ contract CollateralTracker is ERC20Minimal, Multicall {
     /// @notice a token can only be initialized once - which is done by the Panoptic pool.
     bool internal s_initialized;
 
+    /// @dev store whether the current collateral token is token0 of the AMM (true) or token1 (false).
```

Comment:
Please revert this change. The `bool` type is packed as 1 bit in storage, so it will get packed with `s_univ3Token1` and take up the same number of storage slots as what you have changed it to. 
Current: <160 + 1> <160><160+1>|<160>...
Yours: <160 +1 +1><160><160>|<160>...